src/Microsoft.Azure.SignalR.Management/ServiceManager.cs

```diff
@@ -45,7 +45,7 @@ internal ServiceManager(ServiceManagerOptions serviceManagerOptions, string prod
 
             _endpointProvider = new ServiceEndpointProvider(_serverNameProvider, _endpoint, serviceOptions);
 
-            _productInfo = productInfo;
+            _productInfo = serviceManagerOptions.ProductInfo;
```

Comment:
remove it? then input the serviceManagerOptions to RestHubLifetimeManager?

Reply:
@vicancy  As ServiceManagerOptions has some properties not required by RestHubLifetimeManager, I think it would be better to declare exactly what RestHubLifetimeManager needs in its ctor.

Reply:
why put `ProductInfo` into `serviceManagerOptions`? `serviceManagerOptions` sounds like the options for customer to configure, do we want `ProductInfo` to be configured?